lld/ELF/SyntheticSections.cpp

```diff
@@ -2690,6 +2693,756 @@ static uint32_t computeGdbHash(StringRef s) {
   return h;
 }
 
+template <class ELFT>
+DebugNamesSection<ELFT>::DebugNamesSection()
+    : SyntheticSection(0, SHT_PROGBITS, 1, ".debug_names") {}
```

Comment:
I switched to sh_addralign=4 in my branch. I believe the intention is for values in the hash lookup table and the name table to be aligned.